backend/src/app/http/debug.clj

```diff
@@ -433,8 +433,8 @@
 (defn- add-team-feature
   [{:keys [params] :as request}]
   (let [team-id (some-> params :team-id d/parse-uuid)
-        feature (some-> params :feature d/read-string)
-        debug   (contains? params :debug)]
+        feature (some-> params :feature str)
```

Comment:
Minor: the `:feature` is already string so `str` is not necessary,

`(some-> params :feature)` is ok